pylint/lint/run.py

```diff
@@ -412,8 +412,13 @@ def __init__(
                 # We need to make sure we return a failing exit code in this case.
                 # So we use self.linter.msg_status if that is non-zero, otherwise we just return 1.
                 sys.exit(self.linter.msg_status or 1)
-            elif score_value is not None and score_value >= linter.config.fail_under:
-                sys.exit(0)
+            elif score_value is not None:
+                if score_value >= linter.config.fail_under:
+                    sys.exit(0)
+                else:
+                    # We need to make sure we return a failing exit code in this case.
+                    # So we use self.linter.msg_status if that is non-zero, otherwise we just return 1.
```

Comment:
I'm not sure about the ``1`` here. Is it really necessary ? Shouldn't it be 32 because if self.linter.msg_status is 0 then there is only Informational messages ?

Reply:
We do the same on L414. 32 corresponds to `usage error` so that also doesn't make much sense. We don't have a good code for `Information`..

Reply:
Fair enough, it's another issue entirely.